framework/wazuh/core/utils.py

```diff
@@ -1201,16 +1201,25 @@ def _add_limit_to_query(self):
     def _sort_query(self, field):
         return '{} {}'.format(self.fields[field], self.sort['order'])
 
+    def _check_allowed_sort_fields(self, allowed_sort_fields):
+
+        fields_allowed = True
```

Comment:
You don't need to use this variable, just return False inside the if statement and return True after the for loop.